tiles.lua

```diff
@@ -273,10 +331,15 @@ local calculateProperty = {
 
 --===== OBJECTS =====--
 
+--- Object
+-- Each object in this section is a wrapper for a drawable
+-- primitive of the Terminal Glasses API.
+
 local setObjectProperty = {}
 local drawObjectWithType = {}
 local objectMetatables = {}
 
+-- Metaprogramming magic below.
```

Comment:
This is where I'm not sure LDoc is the right approach for this. Because of the way the objects are generated when the API first loads, there is no explicit reference in the code to the individual functions they have. We could just end up making the API file larger without adding any real value...

Do you know a way around this using LDoc?